src/tray/templates/services/app/deploy.yml.j2

```diff
@@ -67,14 +67,18 @@ spec:
             initialDelaySeconds: 10
             periodSeconds: 5
           env:
+            - name: CORS_ALLOWED_ORIGINS
+              value: "{{ joanie_host | richie_host | blue_green_hosts | split(',') | map('regex_replace', '^(.*)$', 'https://\\1') | join(',') }}"
```

Comment:
Are you sure this is working ? What is the result of `joanie_host | richie_host` ?

Reply:
And I'm not sure it is necessary to add joanie itself in the `CORS_ALLOWED_ORIGINS`.

Reply:
Indeed nope, I have to join joanie and richie hosts before passing to `blue_green_hosts` function.

{{ [joanie_host, richie_host] | join(',')}}

Reply:
Yes you are right `joanie_host` is not required here.